packages/qdrant-loader-mcp-server/src/qdrant_loader_mcp_server/fastmcp_tools/expand.py

```
@@ -0,0 +1,112 @@
+def register_expand_tools(mcp: FastMCP) -> None:
+    @mcp.tool(annotations={"readOnlyHint": True})
+    async def expand_document(
+        ctx: Context,
+        document_id: Annotated[
+            str,
+            Field(
+                description="The ID of the document to expand and retrieve full content",
+                min_length=1,
+            ),
+        ],
+        include_metadata: Annotated[
```

Ruff (0.15.20)

[warning] Boolean default positional argument in function definition

(FBT002)